nav2_constrained_smoother/CMakeLists.txt

```diff
@@ -23,13 +23,6 @@ endif()
 
 nav2_package()
 
-set(library_name nav2_constrained_smoother)
```

Comment:
Revert

Reply:
Why is the ${PROJECT_NAME}` variable not used, as is common convention in ROS 2?